pkg/controller/actuator_reconcile.go

```diff
@@ -139,12 +138,6 @@ func (a *actuator) Reconcile(ctx context.Context, _ logr.Logger, network *extens
 		return err
 	}
 
-	timeoutCtx, cancelCtx := context.WithTimeout(ctx, 2*time.Minute)
-	defer cancelCtx()
-	if err := managedresources.WaitUntilHealthy(timeoutCtx, a.client, network.Namespace, CalicoConfigManagedResourceName); err != nil {
-		return err
-	}
```

Comment:
Could you please clarify why you removed this?

Reply:
I added it as per https://github.com/gardener/gardener-extension-networking-calico/pull/293#issuecomment-1689347271, but as explained in https://github.com/gardener/gardener-extension-networking-calico/pull/293#issuecomment-1700483791, the MR cannot be healthy until the deployment is ready.

Reply:
Ok, now I get it. This was only introduced with this PR and then removed again.
Thanks for pointing it out.

Works for me.